### PR TITLE
Add support for Loki Alerting and Recording rules

### PR DESCRIPTION
**Pull request by periklis**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -31,8 +33,12 @@ const (
 // tenantRulesLoader represents logic for loading and filtering PrometheusRule objects by tenants.
 // Useful for testing without spinning up cluster.
 type tenantRulesLoader interface {
+	getLokiAlertingRules() ([]*lokiv1beta1.AlertingRule, error)
+	getLokiRecordingRules() ([]*lokiv1beta1.RecordingRule, error)
 	getPrometheusRules() ([]*monitoringv1.PrometheusRule, error)
-	getTenantRuleGroups(prometheusRules []*monitoringv1.PrometheusRule) map[string]monitoringv1.PrometheusRuleSpec
+	getTenantLokiAlertingRuleGroups(alertingRules []*lokiv1beta1.AlertingRule) map[string]lokiv1beta1.AlertingRuleSpec
```
Comment (saswatamcode) [inline main.go:39]: Maybe this should be same naming pattern as metrics?
```suggestion
	getTenantLogsAlertingRuleGroups(alertingRules []*lokiv1beta1.AlertingRule) map[string]lokiv1beta1.AlertingRuleSpec
```

File: main.go [hunk 2]
```diff
@@ -91,6 +161,8 @@ func (k *kubeRulesLoader) getTenantRuleGroups(prometheusRules []*monitoringv1.Pr
 type tenantRulesSyncer interface {
 	initOrReloadObsctlConfig() error
 	setCurrentTenant(tenant string) error
+	obsctlLogsAlertingSet(rules lokiv1beta1.AlertingRuleSpec) error
```
Comment (saswatamcode) [inline main.go:164]: Seems like some of the methods are a bit repetitive due to split in `lokiv1beta1.AlertingRuleSpec` and `lokiv1beta1.RecordingRuleSpec`. I wonder if we can convert these to `monitoringv1.PrometheusRuleSpec` somehow and unify the methods? 
I think Loki rules are compatible with prometheus afaik. Wdyt? 🙂

File: main.go [hunk 3]
```diff
@@ -42,6 +48,14 @@ type kubeRulesLoader struct {
 	managedTenants string
 }
 
+func (k *kubeRulesLoader) getLokiAlertingRules() ([]*lokiv1beta1.AlertingRule, error) {
+	return nil, nil
```
Comment (saswatamcode) [inline main.go:52]: I guess this is TODO?

**Review by saswatamcode — APPROVED** [review]
Thanks for awesome work! 🚀 

Some tiny suggestions!

File: main.go (hunk 3 above)
Comment (periklis) [reply main.go:52]: Yes I need to figure out a simple way to create a kubeclient w/o import the whole k/k universe here.

**Comment by maorfr** [comment]
❤️

**Review by saswatamcode — APPROVED** [review]
Thanks! LGTM 🙂

**Pull request merged**